src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -350,17 +350,43 @@
     certname]
    (comp first sql/result-set-seq)))
 
+;; `store-catalogs-historically?` is used for toggling historical catalog
+;; storage, the ability to toggle this variable is in the PE extensions repo.
```

Comment:
Seems like we can say this is PE only and configurable and leave out repo information